functions.php

```diff
@@ -505,8 +509,7 @@ function klyp_include_tinypng_lib() {
  * @param    string $width           Width
  * @param    string $height          Height     
  */
-function prepare_resized_image($image_url, $width = 100, $height = 100)
-{
+function prepare_resized_image( $image_url, $width = 100, $height = 100 ) {
```

Comment:
why did you change this line?
the way it was, was fine.

Please revert this change